src/calendar/index.njk

```diff
@@ -2,44 +2,121 @@
 title: Calendar
 layout: layouts/base.njk
 ---
-<div class="grid gap-4">
-  <h2>Events this week</h2>
 
+<div class="grid grid-flow-row gap-3">
+  <h2 class="text-xl mt-5 mx-4">EVENTS THIS WEEK</h2>
+
+  {% set datevariable = "null" %}
```

Comment:
The `null` doesn't need to be quoted.

Can this var name be shorter too? Something like `d` will be great. `{% set d = null %}`